torch/onnx/_internal/fx/type_utils.py

```diff
@@ -30,9 +30,11 @@ def dtype(self) -> Optional[torch.dtype]:
         ...
 
 
-def is_torch_complex_dtype(tensor: TensorLike) -> bool:
+def is_torch_complex_dtype(tensor_or_dtype: TensorLike | torch.dtype) -> bool:
     # NOTE: This is needed as TorchScriptTensor is nor supported by torch.is_complex()
-    return tensor.dtype in _COMPLEX_TO_FLOAT
+    if isinstance(tensor_or_dtype, torch.dtype):
+        return tensor_or_dtype in _COMPLEX_TO_FLOAT
+    return tensor_or_dtype.dtype in _COMPLEX_TO_FLOAT
```

Comment:
```suggestion
    return tensor_or_dtype in _COMPLEX_TO_FLOAT
```

to reduce confusion? i.e. Users don't need to consider whether to use `is_torch_complex_dtype(val)` or `is_torch_complex_dtype.dtype`. Ideally there should only be one correct way.